scripts/generate-merkle-root.ts

```diff
@@ -1,18 +1,72 @@
 import { program } from 'commander'
 import fs from 'fs'
 import { parseBalanceMap } from '../src/parse-balance-map'
+import { keccak256 } from 'ethereumjs-util'
+import { utils } from 'ethers'
+
+const sfc32 = (a: number, b: number, c: number, d: number) => () => {
```

Comment:
We should probably add a comment with the reason why this function is necessary, why we chose `sfc32` and a link to a reference implementation.

Reply:
💯